### PR TITLE
Style and minor functionality update

### PR DESCRIPTION
**Pull request by ismaeldcom**
#19 I start with a simple style & markup update, but I think the sliders event change to see how sliders affect the output in realtime is a good plus.

And the initial sliders output value is just a tiny detail I can't pass 😄 

Isn't a major overhaul, I tried to keep it simple, I you really want to make something different let me know.

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -45,24 +42,17 @@
...
   margin: 10px 0px;
 }
+
 #output-display {
-  height: 100%;
-  text-align: center;
-  padding: 15px auto;
-}
-#output-display canvas {
-  width: 70%;
```
Comment (khutten) [inline css/main.css]: I suspect this width needs to added again to stop the output image from changing size when the ratio is changed.

File: scripts.js [hunk 2]
```diff
@@ -65,6 +68,6 @@ function SetupUpdateEvents(curiosity) {
   const outputDiv = document.getElementById('output-display');
   
   const sliders = document.querySelectorAll('.slider');
-  sliders.forEach(slider => slider.addEventListener('change', updateAfterSliderChange));
```
Comment (khutten) [inline scripts.js]: Curious as to why you changed this from 'change' to 'input'? is that a best practice?

File: css/main.css (hunk 1 above)
Comment (ismaeldcom) [reply css/main.css]: Yes my mistake, I suppose that resize was the functionality, I'll make a new commit.

**Comment by ismaeldcom** [comment]
I just make two new commits, first one to solve the commented problem of resize and the second to limit the maximum width of body and keep web slimmer when the browser is too wide

File: scripts.js (hunk 2 above)
Comment (mrhut10) [reply scripts.js]: @ismaeldcom really someone gem here, i didn't know this. thank you.
really awesome to have you onboard.

File: index.html [hunk 3]
```diff
@@ -8,47 +8,40 @@
...
       </div>
       <div id="output-display">
         <!--output image will get injected here-->
       </div>
     </section>
-    <section id="footer"><h2>
-        footer
-    </h2></section>
+    <section id="footer">
+      <p>A project by <a href="https://github.com/mrhut10">mrhut10</a> and <a href="https://github.com/khutten">khutten</a>.</p>
```
Comment (mrhut10) [inline index.html]: I wonder if we should have a credits page where we have the usernames of anyone who has contributed, could be automated with the github API potentially????
if @khutten @ismaeldcom like this idea i can make a new issue for it, and between us make it a reality.

**Comment by mrhut10** [comment]
really awesome PR thank you @ismaeldcom great to have you onboard.

**Comment by ismaeldcom** [comment]
Thank you, I enjoy contributing to projects differents from my daily work, this is a funny and interesting one.

File: scripts.js (hunk 2 above)
Comment (khutten) [reply scripts.js]: Oh I missed you reply here, Thanks for paying attention to the small things @ismaeldcom! 🙌

File: index.html (hunk 3 above)
Comment (khutten) [reply index.html]: I like the idea, I might make the issue if I get around to it before you.

**Pull request merged**